components/map/MapForPrint.vue

```diff
@@ -1,233 +1,450 @@
+                            </v-row>
+                            <v-row no-gutters align="center">
+                                <v-icon x-small color="#a93130"
+                                    >mdi-square</v-icon
+                                >
+                                <v-col no-gutters cols="6">
+                                    <p>Monitoramento Diário: FF</p>
+                                </v-col>
+                            </v-row>
+                            <v-row no-gutters align="center">
+                                <v-icon x-small color="#ff7f00"
+                                    >mdi-square</v-icon
+                                >
+                                <v-col no-gutters cols="6">
+                                    <p>Monitoramento Diário: DG</p>
+                                </v-col>
+                            </v-row>
+                        </div>
+                        <div v-if="showFeaturesSupportLayers">
+                            <div
+                                v-for="layer in supportLayersCategoryAntropismo"
+                                :key="layer.id"
+                            >
+                                <v-row
+                                    no-gutters
+                                    align="center"
+                                    v-if="layer.visible"
+                                >
+                                    <img
+                                        :src="
+                                            layer.wms.geoserver.preview_url +
+                                            layer.wms.geoserver_layer_name
+                                        "
+                                        width="13vw"
+                                        alt="CorLayer"
+                                    />
+                                    <v-col>
+                                        <p class="ml-1">{{ layer.name }}</p>
+                                    </v-col>
+                                </v-row>
+                            </div>
+                        </div>
+                        <div v-if="showFeaturesSupportLayers">
+                            <div
+                                v-for="layer in supportLayersCategoryFire"
+                                :key="layer.id"
+                            >
+                                <v-row
+                                    no-gutters
+                                    align="center"
+                                    v-if="layer.visible"
+                                >
+                                    <img
+                                        :src="
+                                            layer.wms.geoserver.preview_url +
+                                            layer.wms.geoserver_layer_name
+                                        "
+                                        width="13vw"
```

Comment:
inline style? Why dont you use vue style?

Reply:
Used inline because I don't see the need to create several classes just to define the width of the container. The lib (domtoimage) generates the image according to the size of the elements so it was also necessary to define a fixed width.